README.md

```diff
@@ -38,12 +38,14 @@ Run `make` in the nin folder. This is a convenience method that runs `make` in t
 
 To use grunt, you need to install the command line utility globally, by running `sudo npm install -g grunt-cli`
 
-To make nin available as a global command line utility, the easiest way is to add an alias to your `.bashrc` like this:
+To make nin available as a global command line utility, simply add the path to the backend to your `PATH` in your `.bashrc` file like this:
 
 ```
-alias nin='~/path/to/nin-project/backend/nin'
+export PATH="$PATH:/path/to/nin/nin/backend"
 ```
 
+If you are using Windows, please add this path to your `PATHEXT` variable.
```

Comment:
Ahh my bad, I misread the documentation from shelljs https://github.com/shelljs/shelljs#whichcommand . I'll change this :)